### PR TITLE
Fix github action arm image built

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

: without this, though the docker image is ar,64 arch but  the kmesh-daemon within it is just x86-64.

**Which issue(s) this PR fixes**:
Fixes https://github.com/kmesh-net/kmesh/issues/944

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: build/docker/github.dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM ghcr.io/kmesh-net/kmesh-build:latest AS builder
```
Comment (LiZhenCheng9527) [inline build/docker/github.dockerfile:1]: Why need a new dockerfile?Why not modify the original dockerfile?

Comment (hzxuzhonghu) [reply build/docker/github.dockerfile:1]: This is a temparary fix. Becase that @weli-l found before https://github.com/kmesh-net/kmesh/pull/742, it failed to build using docker 18.09

**Comment by hzxuzhonghu** [comment]
I will merge this manually, since this is influcencing arm deploy

**Pull request merged**